src/SSVOpenHexagon/Core/HGUpdate.cpp

```diff
@@ -50,20 +31,44 @@ void HexagonGame::update(ssvu::FT mFT)
     constexpr float DELAY_TO_UPDATE = 5.f; // X seconds
     timeUntilRichPresenceUpdate -= ssvu::getFTToSeconds(mFT);
 
-    // TODO: show best record (here) and last run + best record (in menu)
-    // Presence formatter
-    const std::string presenceStr =
-        nameStr + " [x" + diffStr + "] - " + timeStr + "s";
-
     if(timeUntilRichPresenceUpdate <= 0.f)
     {
         steamManager.set_rich_presence_in_game(nameStr, diffStr, timeStr);
-        discordManager.set_rich_presence_in_game(presenceStr);
+
         timeUntilRichPresenceUpdate = DELAY_TO_UPDATE;
     }
 
-    steamManager.run_callbacks();
-    discordManager.run_callbacks();
+    // Update Steam Rich Presence
```

Comment:
These two blocks of code warrant their own `private` member functions.

Reply:
I will make the callbacks their own function, but it has to be public because MenuGame has to run this function as well.